content/blog/2024/how-to-write-stellar-proposals.md

```diff
@@ -0,0 +1,89 @@
+</span>
+
+### Diverse Audience:
+
+Your insights and experiences with Python will reach a broad and engaged audience, encompassing everyone from beginners to expert developers. PyCon India thrives on the diversity of its participants and their contributions.
+
+### Mentorship for New Speakers:
+
+New to speaking at conferences? No problem! PyCon India offers a robust mentorship program to help you prepare. Experienced speakers from our community are here to guide you through crafting and delivering your talk or workshop, ensuring you feel ready and confident.
+
+### Opportunities to Contribute and Grow:
+
+Speaking at PyCon India is more than just a presentation; it's an opportunity to contribute to the Python community's growth. You'll receive valuable feedback, inspire others, and network with Python experts, fostering innovation.
+
+
+## How to write Stellar Proposals?
+
+**Craft an informative and persuasive title that serves as an elevator pitch:**
+Your title is your talk's elevator pitch, shaping the first impression for both the selection committee and the audience. It will be alongside numerous other proposals, influencing track selection on the conference day.
+
+**Provide detailed descriptions outlining the problem, solution, and audience takeaways:**
+Ensure your detailed description addresses the problem, your approach to solving it, and the audience takeaway. Use it to showcase your expertise to the selection team. Consider adding an outline at the end, highlighting the topics and their duration.
+
+
+**Attach slides, if available:**
+Adding more details to your proposal enhances clarity for the selection committee. Include slides, if available, even in skeleton form, to provide additional context.
+
+**Emphasize your expertise in the speaker bio, showcasing relevant experience:**
+Leverage the speaker bio to showcase your qualifications and why you're suited to deliver the talk. It is completely okay if you are yet to gain speaking experience. 
+
+**Include a two-minute casual video:** Showcase your presentation style and link it to the proposal. This will strengthen your chances of selection. 
+
+
+### Here are some tips to enhance the proposal and make it stand out:
+
+- Make your proposal unique by offering fresh insights, innovative solutions, and original perspectives. Avoid repeating previous ideas and plagiarism. 
+- Tailor your content to diverse audience levels and interests. 
+- Demonstrate your expertise in the topic with evidence of contributions to the Python community. Mention relevant publications, talks, or projects you would have worked on. 
+- Proofread your proposal for errors and clarity before submission.
+- Ensure that your talk content does not appear promotional. Keep it neutral and objective, discussing both commercial and open-source projects. Cover alternative perspectives and consider all sides of the topic. Use neutral language and avoid bias.
+- Adhere to the conference's code of conduct.
```

Comment:
Might be worthwhile linking to the CoC here?

Reply:
```suggestion
- Adhere to the [conference's code of conduct](https://in.pycon.org/2024/code-of-conduct/).
```